config/index.conf.php

```diff
@@ -35,7 +44,7 @@
             "match" => "TextArea_*",
             "match_mapping_type" => "string",
             "mapping" => [
-                "type" => "text"
```

Comment:
As far as I know, this would prevent partial matching on text area fields, right? ([ref #1](https://stackoverflow.com/a/69282029/4399063), [ref #2](https://kb.objectrocket.com/elasticsearch/when-to-use-the-keyword-type-vs-text-datatype-in-elasticsearch))

Reply:
It still supports, I tested now.